tests/integration/tests/cluster/test-cluster-configuration.spec.js

```diff
@@ -74,6 +74,11 @@ context('Test Cluster configuration', () => {
 
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```

Comment:
do we need this `.invoke('show')` for the test to pass?
at least locally it works for me without it

Reply:
Form me without `.invoke('show')` it couldn't find `li`

Reply:
Yesterday without `.invoke('show')` it failed at finding `li`, but today it somehow work. So I'll remove all the `.invoke('show')`